addons/website_links/tests/test_link_tracker.py

```diff
@@ -0,0 +1,77 @@
+from odoo.tests import TransactionCase, tagged
+from odoo.addons.mail.tests.common import mail_new_test_user
+
+
+@tagged('post_install', '-at_install')
+class TestLinkTracker(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.company_1 = cls.env['res.company'].create({
+            'name': 'Company 1',
+        })
+
+        cls.company_2 = cls.env['res.company'].create({
+            'name': 'Company 2',
+        })
+
+        cls.website_1, cls.website_2 = cls.env['website'].create([
+            {
+                'name': 'Website 1',
+                'domain': 'https://maincompany.odoo.com',
+                'company_id': cls.company_1.id
+            },
+            {
+                'name': 'Website 2',
+                'domain': 'https://secondarycompany.odoo.com',
+                'company_id': cls.company_2.id
+            }
+        ])
+
+        cls.company_1.write({'website_id': cls.website_1.id})
+        cls.company_2.write({'website_id': cls.website_2.id})
+
+        cls.test_user = mail_new_test_user(
+            cls.env,
+            login='test_user',
+            name='Test User',
+            company_id=cls.company_1.id,
+            company_ids=[(6, 0, [cls.company_1.id, cls.company_2.id])]
+        )
+
+    def setUp(self):
+        super().setUp()
+        self.env.user.company_id = self.company_1
```

Comment:
That's not what I meant :sweat_smile: use odoo data companyu as first company so that you just have to create another one. Aka: something like
```
        cls.company_1 = cls.env.company
        cls.company_2 = cls.env['res.company'].create ...
```